caktus/etlman
# 41 navbar projects

Adds `Projects` tab to the navbar, filters Projects where the currently-logged in user is a/the collaborator, and includes a link to the `Add a project` page.

**To test:**

1. Create some projects either via the admin or using the web UI
2. Ensure that the existing projects display in the tab
3. Log in as a different user (a user that does not share the same projects) and ensure that the projects do not display
4. Click the add a project link and ensure it takes you to the right page

Closes #41
